src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const liElems = document.querySelectorAll('li');
+const list = document.querySelector('.tree');
+
+for (const item of liElems) {
+  const spanEl = document.createElement('span');
+
+  item.prepend(spanEl);
+  spanEl.append(spanEl.nextSibling);
+}
+
+list.addEventListener('click', (evnt) => {
+  const item = evnt.target;
+
+  if (item.nextSibling === null) {
+    return;
+  }
+
+  if (item.nextSibling.style.display === 'none') {
+    item.nextSibling.style.display = 'block';
+
+    return;
+  }
+
+  item.nextSibling.style.display = 'none';
```

Comment:
```suggestion
  el.target.nextSibling.hidden = !el.target.nextSibling.hidden;
```
you can use hidden property